botCommands/club_40_gifs.json

```diff
@@ -63,4 +63,8 @@
     "gif": "https://i.imgur.com/DVrRqoF.gif",
     "author": "Sully"
   }
```

Comment:
@rishikumars-in We missed a comma here which brought the bot down. I missed it in my review. It has been fixed.
No big deal. Good learning moment!